docs/mobile/overview.mdx

```diff
@@ -15,75 +16,58 @@ Geocortex Mobile was designed to be highly configurable out of the box. Geocorte
 
 <DesignerCallout />
 
-## Start with the Concepts
+## Extending Geocortex Mobile
 
-The core element of Geocortex Mobile is an **Application**. Applications are composed of three main pieces:
+If the Geocortex Mobile Designer cannot accomplish what you need, you can manually configure and extend Geocortex Mobile in a number of ways.
 
-1. [**Layout XML**](configuration-layout-getting-started.mdx) that represents the visual layout of the application.
+The simplest way to customize your application is through configuration. If you need to implement complex behavior and user interactions, Geocortex Workflow may be able to accomplish your goals through an easy to use interface. Geocortex Workflow can be taken even further through custom extensions to its activity and form element library. Geocortex Mobile can also be extended through its SDK to create completely custom experiences.
 
-2. [**App Config JSON**](configuration-app-config-getting-started.mdx) that configures the application.
+<ExtensibilitySpectrum
+    configLink="#application-configuration"
+    workflowLink="#geocortex-workflow"
+    customWorkflowLink="#custom-workflow-activities-and-form-elements"
+    customCodeLink="#custom-code"
```

Comment:
Heads up these links look to be incorrect after we renamed the heading in both overviews.

Reply:
Good catch! Fixed.